mkosi/distributions/fedora.py

```diff
@@ -74,12 +102,8 @@ def remove_packages(cls, context: Context, packages: Sequence[str]) -> None:
     @classmethod
     @listify
     def repositories(cls, context: Context) -> Iterable[RpmRepository]:
-        gpgurls = (
-            find_rpm_gpgkey(
-                context,
-                key=f"RPM-GPG-KEY-fedora-{context.config.release}-primary",
-            ) or "https://fedoraproject.org/fedora.gpg",
-        )
+        gpgurls = find_fedora_rpm_gpgkeys(context)
```

Comment:
I think I'd forego introducing `tuplify` and just use `listify` and just call `tuple` on it here.

Reply:
I think it's important to hide such ugly details… You define `tuplify` once and then can have nicer code in multiple places.